packages/graphile-utils/src/makePgSmartTagsPlugin.ts

```diff
@@ -240,7 +240,7 @@ export type JSONPgSmartTags = {
   };
 };
 
-function pgSmartTagRulesFromJSON(
+export function pgSmartTagRulesFromJSON(
```

Comment:
You also need to export this from `index.ts`

Reply:
Doh. Does `pgSmartTagRulesFromJSON` get exported when we `export * from "./makePgSmartTagsPlugin";` in [graphile-utils/src/index.ts#L22](https://github.com/graphile/graphile-engine/blob/9940895db5a35f218484620d610845201278611c/packages/graphile-utils/src/index.ts#L22)?

Reply:
So it does; I generally have a policy of not using `export *` (because it doesn't work well with TypeScript declaration merging) so wasn't expecting that. :man_shrugging: